src/components/staking/PageValidator.vue

```diff
@@ -308,9 +310,15 @@ export default {
     }
   }),
   computed: {
-    ...mapState([`connection`]),
+    ...mapState([`connection`, `session`]),
     ...mapGetters([`network`, `stakingDenom`]),
-    ...mapGetters({ userAddress: `address` })
+    ...mapGetters({ userAddress: `address` }),
+    isRestrictedAddressRole() {
```

Comment:
why?

Reply:
xD

Reply:
following my instructions the user opens the action modal and has the option to do the action he can do

Reply:
in PageValidator he can do nothing

Reply:
a controller can do staking
and he should be able to open the ActionModal in any case